db/migrate/20221115082403_add_ldap_tls_options.rb

```diff
@@ -41,10 +41,27 @@ def change
       dir.up do
         # Current LDAP library default is to not verify the certificate
         MigratingAuthSource.reset_column_information
-        ldap_settings = (Setting.ldap_tls_options || {}).with_indifferent_access
-        verify_peer = ldap_settings[:verify_mode] == OpenSSL::SSL::VERIFY_PEER
-        MigratingAuthSource.update_all(verify_peer:)
+
+        ldap_settings = Setting.find_by(name: 'ldap_tls_options')&.value
+        migrate_ldap_settings(ldap_settings)
+
+        Setting.where(name: 'ldap_tls_options').delete_all
```

Comment:
Since this is an existing migration, it will be not ran for instances who are upgrading to the next version.
Meaning, the `ldap_tls_options` settings will remain in place and won't be deleted. This is not necessarily a problem, because we do not expose the setting anymore, but it's gonna be a dangling setting.

Reply:
Indeed that's a good point. I'll move it to a separate migration